pkr/__main__.py

```diff
@@ -20,18 +20,21 @@
 def main():
     """Main function"""
     try:
-        cli_args = get_parser().parse_args()
+        parser = get_parser()
+        cli_args = parser.parse_args()
 
         # Setting the log mode
-        debug = cli_args.__dict__.pop('debug')
+        debug = getattr(cli_args, 'debug')
```

Comment:
I don't understand the `__dict__.pop` or the getattr. The usually and the KISS method is
`cli_args.debug`

Reply:
I think the __dict__.pop was to remove the attribute at the same time